plugins/UsersManager/API.php

```diff
@@ -802,22 +992,31 @@ public function getUserLoginFromUserEmail($userEmail)
      * If access = 'noaccess' the current access (if any) will be deleted.
      * If access = 'view' or 'admin' the current access level is deleted and updated with the new value.
      *
-     * @param string $userLogin The user login
+     * @param string|string[] $userLogin The user login(s)
```

Comment:
Same here can we use Bulk API requests instead of allowing multiple logins? Complexity is getting quite high and chances of (security) bugs high.